js/app.js

```diff
@@ -1,122 +1,132 @@
+          $(this).fadeIn('fast');
+        }
+      });
+      $('.mobilenav').fadeOut(500);
+      $('.top-menu').removeClass('top-animate');
+      $('body').removeClass('noscroll');
+      $('.mid-menu').removeClass('mid-animate');
+      $('.bottom-menu').removeClass('bottom-animate');
     });
-    $('.mobilenav').fadeOut(500);
-    $('.top-menu').removeClass('top-animate');
-    $('body').removeClass('noscroll');
-    $('.mid-menu').removeClass('mid-animate');
-    $('.bottom-menu').removeClass('bottom-animate');
-  });
-};
 
-// Handle JSON
-Project.loadAll = function(dataWePassIn) {
-  JSON.parse(dataWePassIn).forEach(function(ele) {
-    projects.push(new Project(ele));
-  });
-};
+    $('#linkedin').on('touchstart', function(){
+      window.location = 'https://www.linkedin.com/in/jeffrey-gebhardt-b1976451';
+    });
+    $('#github').on('touchstart', function(){
+      window.location = 'https://github.com/jeffgebhardt';
+    });
+  };
 
-Project.appendAll = function(dataToAppend) {
-  dataToAppend.forEach(function(p){
-    $('#projects').append(p.toHtml());
-  });
-};
+  // Handle JSON
+  Project.loadAll = function(dataWePassIn) {
+    projects = JSON.parse(dataWePassIn).map(function(ele){
+      return new Project(ele);
+    });
+  };
 
-Project.fetchAll = function(){
-  if (localStorage.data) {
-    console.log('Local Storage DOES Exist');
+  Project.appendAll = function(dataToAppend) {
+    dataToAppend.map(function(p) {
+      return $('#projects').append(p.toHtml());
+    });
+  };
 
-    $.ajax({
-      url: 'data/projects-data.json',
-      success: function (data, message, xhr) {
-        var eTag = xhr.getResponseHeader('eTag');
-        console.log('Current eTag = ' + eTag);
-        var compareETag = JSON.parse(localStorage.getItem('eTag1'));
-        console.log('Compare eTag = ' + compareETag);
-        // console.log(message, eTag);
-        if (eTag !== compareETag) {
-          console.log('Changes to JSON file detected');
-          console.log('Retrieved data from JSON file...');
-          Project.loadAll(JSON.stringify(data));
-          Project.appendAll(projects);
-          //Set Local Storage
-          localStorage.setItem('data', JSON.stringify(projects));
-          console.log('Setting Local Data...');
+  Project.fetchAll = function(){
+    if (localStorage.data) {
+      console.log('Local Storage DOES Exist');
+
+      $.ajax({
+        type: 'head',
+        url: 'data/projects-data.json',
+        success: function (data, message, xhr) {
+          var eTag = xhr.getResponseHeader('eTag');
+          console.log('Current eTag = ' + eTag);
+          var compareETag = JSON.parse(localStorage.getItem('eTag1'));
+          console.log('Compare eTag = ' + compareETag);
+
+          if (eTag !== compareETag) {
+            console.log('Changes to JSON file detected');
+            console.log('Retrieved data from JSON file...');
+            Project.loadAll(JSON.stringify(data));
```

Comment:
You shouldn't need to stringify the data here, that might actually cause problems.